public/js/monitor.js

```diff
@@ -510,6 +510,35 @@ async function initVpnBanner() {
   }
 }
 
+async function initCsatBanner() {
+  const csatBanner = document.querySelector(".csat-banner");
+
+  if (!csatBanner) return;
+
+  csatBanner.addEventListener("click", handleEvent);
+
+  function handleEvent(e) {
+    const ttl = new Date();
+    ttl.setDate(ttl.getDate() + 90);
+    document.cookie = "csatHidden=1; path=/; sameSite=Lax; expires=" + ttl.toUTCString();
+
+    switch (e.target.name) {
+      case "csat-close-btn":
+        csatBanner.toggleAttribute("hidden", true);
+        csatBanner.removeEventListener("click", handleEvent);
```

Comment:
question: could we add an event for `CSAT banner` `close` so we can keep track of how many `close` events we're getting compared to `submit` events?

Reply:
This was added but then removed to avoid complicating the result set.  It's not a good measure of how many people dismiss the survey, because they can just scroll past.  Also, a naive approach would have tracked closing the banner _after_ form submission, as well as before – making the metric even less useful.